modules/dailymotionBidAdapter.md

```diff
@@ -122,15 +139,29 @@ If a field exists in both places, it will be overridden by bids.params.video.
 
```

Comment:
Please edit the following statement:

> Each of the following video metadata fields can be added in mediaTypes.video or bids.params.video.

Only fields supported in `imp[].video` should be defined in `mediaTypes.video`